serverless.yml

```diff
@@ -262,6 +263,26 @@ functions:
               - name: request.querystring.search_after
               - name: request.header.origin
 
+  get_block_with_biggest_height:
+    handler: handlers/block_api.get_block_with_biggest_height
+    timeout: 15
+    maximumRetryAttempts: 0
+    package:
+      patterns:
+        - 'handlers/block_api.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: blocks/biggest_height
```

Comment:
Wouldn't something like `blocks/bestchain_height` or `blocks/current_height` be more explicit of what the api returns?

Reply:
I agree. Changed to `blocks/bestchain_height`